README.md

```diff
@@ -1,61 +1,110 @@
-- Create one or more JavaScript files which contain extensions of the page grades (e.g. for `sjrk.storyTelling.page.storyEdit`, add a new grade `sjrk.storyTelling.myCustomTheme.storyEdit` which has the former as one of its gradeNames)
-- Add any new associated handlebars templates to be used by each new page
-- If your new template file requires new message values (i.e. new wording), you can create new message bundle files or add them to the existing message files. Due to the nature of ui grades, each template may use only one message file at a time, so if new messages are added to an extended page, those may have to be duplicated in a new message file.
-- If there are some common changes that will be made to all pages, or shared values, create a 'base' page grade which includes these common changes and have every new page refer to that gradeName as well. For example:
-    - `sjrk.storyTelling.myCustomTheme.storyEdit` could have the following as a value for gradeNames: `["sjrk.storyTelling.myCustomTheme", "sjrk.storyTelling.page.storyEdit"]`
-    - if the folder structure for the new theme is different from the original location of these files, you could update the `pageSetup.resourcePrefix` option of each new page by putting it in the new base grade
-- Add any custom assets required, such as new images, sounds, or fonts.
-- Create a common CSS file with all of the rules for the new theme
-- Add a link in each new HTML file for any applicable new CSS or JavaScript files
-- For each new HTML file, adjust the page component's creation function to use the new custom component
-    - e.g. update `sjrk.storyTelling.page.storyEdit` to `sjrk.storyTelling.myCustomTheme.storyEdit`
-
-### Licenses
-The SJRK Storytelling Tool is provided under either the New BSD license or the Educational Community License, Version 2.0. Please see LICENSE.txt for more details.
+Custom themes can be added to the Storytelling Tool by a process detailed in the documentation of the UI project. If you have a set of custom theme grades and files that you would like to use, you need to:
+- In the part of storyTellingServerUI.js that defines the `templates` variable, add an entry for your new custom theme that contains any DOM elements that are required to inject markup or templates.
+    - e.g.
+        ```javascript
+        myCustomTheme: {
+            view: '<div class="sjrk-pageBody-container sjrk-pageBody-container-oneColumn"> YOUR CUSTOM CONTAINERS GO HERE </div>',
+            edit: '<div class="sjrk-pageBody-container sjrk-pageBody-with-sidebars"> YOUR CUSTOM CONTAINERS GO HERE </div>',
+            browse: '<div class="sjrk-pageBody-container sjrk-pageBody-container-oneColumn"> YOUR CUSTOM CONTAINERS GO HERE</div>'
+        }
+        ```
+- In each of the HTML files in the `ui` directory, update the call to `sjrk.storyTelling.server.loadThemedPage` to use your desired theme
```

Comment:
I noticed `sjrk.storyTelling.server.loadThemedPage` is now renamed to `sjrk.storyTelling.loadThemedPage` (the removal of `server` from the namespace).